.pre-commit-config.yaml

```diff
@@ -1,12 +1,30 @@
 ---
+exclude: '^.*\.pem$'
+
 repos:
-  - repo: https://github.com/psf/black
-    rev: stable
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.4.0
     hooks:
-      - id: black
-        args: [--line-length=100]
+      - id: no-commit-to-branch
+        args: [--branch, dev, --branch, master]
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
+        args: [--fix=lf]
+      - id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+      - id: check-added-large-files
+      - id: check-executables-have-shebangs
+      - id: check-merge-conflict
+      - id: check-docstring-first
+      - id: check-yaml
 
   - repo: https://github.com/pre-commit/mirrors-isort
     rev: v4.3.21
     hooks:
       - id: isort
+
+  - repo: https://github.com/psf/black
+    rev: stable
```

Comment:
Can we actually pin this to a specific version so we don't have to worry about someone getting malicious code into this repo?